### ISSUE TITLE
an example fails in first_long.py

### ISSUE THREAD

**Issue by videlec** [Normaliz/PyNormaliz#26]
- testing `examples/first_long.py` produces
```
Traceback (most recent call last):
  File "first_long.py", line 18, in <module>
    cone = PyNormaliz.NmzCone(["equations", eq,"grading",grading],CreateAsLongLong=True)
Normaliz.interface_error: Even entries must be matrices
```

**Comment by videlec** [comment]
Mostly fixed by  997ae34

**Comment by sebasguts** [comment]
Should be fixed in ae6aacc2b90811ad837b4cc2028836f22e097037
